sdcm/cluster_aws.py

```diff
@@ -339,25 +341,33 @@ def update_bootstrap(ec2_user_data, enable_auto_bootstrap):
                 ec2_user_data += ' --bootstrap false '
         return ec2_user_data
 
-    def _create_or_find_instances(self, count, ec2_user_data, dc_idx, az_idx=0):
-        if self.nodes:
-            return self._create_instances(count, ec2_user_data, dc_idx, az_idx)
+    # pylint: disable=too-many-arguments
+    def _create_or_find_instances(self, count, ec2_user_data, dc_idx, az_idx):
         if self.test_config.REUSE_CLUSTER:
-            instances = self._get_instances(dc_idx)
+            instances = self._get_instances(dc_idx, running=True)
```

Comment:
this line is worth commiting, as currently we could have issue with decommissioned nodes be taken up.